slack-channel.el

```diff
@@ -39,6 +39,13 @@
 (defclass slack-channel (slack-group)
   ((is-member :initarg :is_member :initform nil :type boolean)))
 
+(cl-defmethod slack-room-name ((room slack-channel) team)
+  (if (slack-mpim-p room)
+      (format "MPIM: %s"
+              (s-join ", " (mapcar (lambda (userid)
```

Comment:
I would recommend using `string-join` function here instead of `s-join` as `s.el` is not a dependency of `emacs-slack` right now.

Reply:
Done. Thank you